### PR TITLE
PIM-4838: add hasSelectedLocale()

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

File: features/Context/Page/Base/Base.php [hunk 1]
```diff
@@ -21,13 +21,14 @@ class Base extends Page
...
-        'Navigation Bar' => ['css' => 'header#oroplatform-header'],
-        'Container'      => ['css' => '#container'],
+        'Dialog'           => ['css' => 'div.modal'],
+        'Title'            => ['css' => '.navbar-title'],
+        'Product title'    => ['css' => '.product-title'],
+        'HeadTitle'        => ['css' => 'title'],
+        'Flash messages'   => ['css' => '.flash-messages-holder'],
+        'Navigation Bar'   => ['css' => 'header#oroplatform-header'],
+        'Container'        => ['css' => '#container'],
+        'Locales dropdown' => ['css' => '#locale-switcher'],
```
Comment (sumbobyboys) [inline features/Context/Page/Base/Base.php]: you are not responsible of that, but we should not rely on an id here ..

**Comment by phaseinducer** [comment]
GTM

File: composer.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
         "monolog/monolog": "1.10.0",
         "nelmio/api-doc-bundle": "2.9.0",
         "oneup/flysystem-bundle": "1.1.0",
-        "oro/platform": "0.10.0",
+        "oro/platform": "dev-PIM-4838",
```
Comment (jjanvier) [inline composer.json]: TO REVERT

**Comment by jjanvier** [comment]
GTM once fixed

**Pull request merged**